libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -74,17 +73,9 @@ class RouterWrapperTests {
 
     private val testRouteFixtures = TestRouteFixtures()
 
-    private val routerResultSuccess: Expected<RouterError, String> = mockk {
-        every { isValue } returns true
-        every { isError } returns false
-        every { value } returns testRouteFixtures.loadTwoLegRoute()
-        every { error } returns null
-
-        val valueSlot = slot<Expected.Transformer<String, Unit>>()
-        every { fold(any(), capture(valueSlot)) } answers {
-            valueSlot.captured.invoke(this@mockk.value!!)
-        }
-    }
+    private val routerResultSuccess: Expected<RouterError, String> = ExpectedFactory.createValue(
+        testRouteFixtures.loadTwoLegRoute()
+    )
```

Comment:
@RingerJK , @dzinad, do you feel that it's an abuse of mockk? Don't you mind if I replace all mocks of `Expected` in this test?

Reply:
I think it depends on if we want to test a real road our just react of result somehow

Reply:
The mock uses a real route. See line 80. It just replicates behaviour of `Expected`

Reply:
I guess that for so simple a class as expected it's easier and more convenient to use the real object. Actually, you can pass mocked value if the value type is too complex. So the approaches can be easily mixed.